### PR TITLE
Update documentation after package migration

### PR DESCRIPTION
**Pull request by lchico**
## Description
|Related issue|
|---|
https://github.com/wazuh/wazuh-documentation/issues/7184
||

### Docs building
- [ ] Compiles without warnings.
### Code formatting and web optimization
- [x] Uses three spaces indentation.
- [x] Adds or updates meta descriptions accordingly.
- [x] Updates the `redirects.js` script if necessary (check [this guide](https://github.com/wazuh/wazuh-documentation/blob/master/NEW_RELEASE.md)).
### Writing style
- [x] Uses present tense, active voice, and semi-formal registry.
- [x] Uses short, simple sentences.
- [x] Uses **bold** for user interface elements, _italics_ for key terms or emphasis, and `code` font for Bash commands, file names, REST paths, and code.

### CODE REVIEW

File: source/development/packaging/generate-deb-rpm-package.rst [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+To create an Debian or RPM package follow these steps:
+
+Requirements
+^^^^^^^^^^^^
+
+ * Docker
+ * Git
+
+Download our wazuh repository from GitHub and go to the debs directory.
```
Comment (MarcelKemp) [inline source/development/packaging/generate-deb-rpm-package.rst:21]:
```suggestion
Download our wazuh repository from GitHub and go to the packages directory.
```

File: source/development/packaging/generate-osx-package.rst [hunk 2]
```diff
@@ -39,12 +39,15 @@ Execute the ``generate_wazuh_packages.sh`` script, with the different options yo
   Usage: ./generate_wazuh_packages.sh [OPTIONS]
 
     Build options:
-      -a, --architecture <arch>     [Optional] Select architecture to build (intel64 or arm64). Default: intel64
-      -b, --branch <branch>         [Required] Select Git branch or tag.
+      -a, --architecture <arch>     [Optional] Target architecture of the package [intel64/arm64]. By Default: intel64.
+      -b, --branch <branch>         [Optional] Select Git branch [].
```
Comment (MarcelKemp) [inline source/development/packaging/generate-osx-package.rst:43]:
```suggestion
      -b, --branch <branch>         [Optional] Select Git branch.
```

**Review by MarcelKemp — CHANGES_REQUESTED** [review]
GJ, just a few minor changes.

File: source/development/packaging/generate-deb-rpm-package.rst (hunk 1 above)
Comment (mjcr99) [reply source/development/packaging/generate-deb-rpm-package.rst]: Fixed by https://github.com/wazuh/wazuh-documentation/pull/7204/commits/3c3c3c0809054b5d51d713c6668fe71ea8525c21.

File: source/development/packaging/generate-osx-package.rst (hunk 2 above)
Comment (mjcr99) [reply source/development/packaging/generate-osx-package.rst]: Fixed by https://github.com/wazuh/wazuh-documentation/pull/7204/commits/3c3c3c0809054b5d51d713c6668fe71ea8525c21.

**Review by MarcelKemp — DISMISSED** [review]

**Review by MarcelKemp — APPROVED** [review]

File: source/development/packaging/generate-aix-package.rst [hunk 3]
```diff
@@ -5,8 +5,9 @@
   
 .. _create-aix:
 
-AIX
-===
+=========
```
Comment (javimed) [inline source/development/packaging/generate-aix-package.rst:8]:
```suggestion
```

Comment (javimed) [reply source/development/packaging/generate-aix-package.rst:8]: To keep consistency with other files, please remove overlining which is just optional. Same comment for the other occurrences.

File: source/development/packaging/generate-deb-package.rst
Comment (javimed) [inline source/development/packaging/generate-deb-package.rst:1]: Please add an entry in removedUrls in `redirects.js` file. For example

```
removedUrls['4.9'] = [
  '/deployment-options/offline-installation.html',
  '/development/packaging/generate-deb-package.html',
];
```

Add a redirection between 4.8 and 4.9 versions in redirections.push for this file in `redirects.js` as well. For example

```
/* Redirections from 4.8 to 4.9  */

redirections.push(
  ...
  {
    'target': ['4.8=>4.9'],
    '4.8': '/development/packaging/generate-deb-package.html',
    '4.9': '/development/packaging/generate-deb-rpm-package.html',
  },
  ...
);

/* Pages added in 4.9 */
```

File: source/development/packaging/generate-rpm-package.rst
Comment (javimed) [inline source/development/packaging/generate-rpm-package.rst:1]: Same as before. Please update `redirects.js` with file removal and page redirection.

File: source/development/packaging/generate-deb-rpm-package.rst
Comment (javimed) [inline source/development/packaging/generate-deb-rpm-package.rst:1]: Please add an entry in newUrls in `redirects.js` file.

**Review by javimed — CHANGES_REQUESTED** [review]

**Pull request merged**